conf/mesos-site.xml

```diff
@@ -99,12 +109,12 @@
 
   <property>
     <name>mesos.hdfs.journalnode.cpus</name>
-    <value>1</value>
+    <value>0.5</value>
```

Comment:
No work has been done! I'm working on integration tests. For now, I would err on the side of small.